<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camdisplay</class>
 <widget class="QWidget" name="camdisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">outline: none;
color: rgb(255, 255, 255);
background-color: rgb(62, 63, 63);</string>
  </property>
  <widget class="QWidget" name="configuration" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>251</width>
     <height>871</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: none;
color: rgb(255, 255, 255);
background-color: rgb(42, 43, 43);</string>
   </property>
   <widget class="QListWidget" name="CamListWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
     <item>
      <widget class="QLabel" name="Cameradeviceslabel">
       <property name="styleSheet">
        <string notr="true">color:white; font-size:15px</string>
       </property>
       <property name="text">
        <string>Devices:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Devices">
       <property name="styleSheet">
        <string notr="true">min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);</string>
       </property>
       <item>
        <property name="text">
         <string>InterRealSense</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white; font-size:15px</string>
    </property>
    <property name="text">
     <string>Devices Description:</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="abovewidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(135, 155, 170);</string>
   </property>
   <widget class="QPushButton" name="D2">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>0</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D3">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>0</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>3D</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>0</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Start" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>1081</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline: none;
color: rgb(255, 255, 255);
background-color: rgb(42, 43, 43);</string>
   </property>
   <widget class="QPushButton" name="run_all">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius:4px;
border-color: rgb(35, 143, 159);
background-color: rgb(70, 70, 83);
}
QPushButton:pressed{
border-radius:5px;border:4px solid gray;
background-color: rgb(90, 90, 103);
border-color: rgb(35, 143, 159);
color: rgb(255, 255, 255);
}
QPushButton:disabled{
border-radius:5px;
border:2px solid gray;
background-color: rgb(20, 30, 13);
border-color: rgb(15, 103, 109);
color: rgb(205, 205, 205);
border-style: outset;
padding: 1px;
}</string>
    </property>
    <property name="text">
     <string>Run all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius:4px;
border-color: rgb(35, 143, 159);
background-color: rgb(70, 70, 83);
}
QPushButton:pressed{
border-radius:5px;border:4px solid gray;
background-color: rgb(90, 90, 103);
border-color: rgb(35, 143, 159);
color: rgb(255, 255, 255);
}
QPushButton:disabled{
border-radius:5px;
border:2px solid gray;
background-color: rgb(20, 30, 13);
border-color: rgb(15, 103, 109);
color: rgb(205, 205, 205);
border-style: outset;
padding: 1px;
}</string>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>0</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius:4px;
border-color: rgb(35, 143, 159);
background-color: rgb(70, 70, 83);
}
QPushButton:pressed{
border-radius:5px;border:4px solid gray;
background-color: rgb(90, 90, 103);
border-color: rgb(35, 143, 159);
color: rgb(255, 255, 255);
}
QPushButton:disabled{
border-radius:5px;
border:2px solid gray;
background-color: rgb(20, 30, 13);
border-color: rgb(15, 103, 109);
color: rgb(205, 205, 205);
border-style: outset;
padding: 1px;
}</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="detect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius:4px;
border-color: rgb(35, 143, 159);
background-color: rgb(70, 70, 83);
}
QPushButton:pressed{
border-radius:5px;border:4px solid gray;
background-color: rgb(90, 90, 103);
border-color: rgb(35, 143, 159);
color: rgb(255, 255, 255);
}
QPushButton:disabled{
border-radius:5px;
border:2px solid gray;
background-color: rgb(20, 30, 13);
border-color: rgb(15, 103, 109);
color: rgb(205, 205, 205);
border-style: outset;
padding: 1px;
}</string>
    </property>
    <property name="text">
     <string>Detect</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius:4px;
border-color: rgb(35, 143, 159);
background-color: rgb(70, 70, 83);
}
QPushButton:pressed{
border-radius:5px;border:4px solid gray;
background-color: rgb(90, 90, 103);
border-color: rgb(35, 143, 159);
color: rgb(255, 255, 255);
}
QPushButton:disabled{
border-radius:5px;
border:2px solid gray;
background-color: rgb(20, 30, 13);
border-color: rgb(15, 103, 109);
color: rgb(205, 205, 205);
border-style: outset;
padding: 1px;
}</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_all">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
min-width: 60px; min-height: 22px;
text-align: center;
border: 2px solid gray;
color: rgb(255, 255, 255);
border-radius:5px;
border-color: rgb(35, 143, 159);
background-color: rgb(50, 50, 63);
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius:4px;
border-color: rgb(35, 143, 159);
background-color: rgb(70, 70, 83);
}
QPushButton:pressed{
border-radius:5px;border:4px solid gray;
background-color: rgb(90, 90, 103);
border-color: rgb(35, 143, 159);
color: rgb(255, 255, 255);
}
QPushButton:disabled{
border-radius:5px;
border:2px solid gray;
background-color: rgb(20, 30, 13);
border-color: rgb(15, 103, 109);
color: rgb(205, 205, 205);
border-style: outset;
padding: 1px;
}</string>
    </property>
    <property name="text">
     <string>Stop all</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gap" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>20</width>
     <height>871</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);</string>
   </property>
  </widget>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>1061</width>
     <height>811</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
